webapp/src/components/AuctionFinishedPage/AuctionFinishedPage.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Button, Loader } from 'semantic-ui-react'
+import { t } from '@dapps/modules/translation/utils'
+
+import { formatMana } from 'lib/utils'
+import { AUCTION_DURATION_IN_DAYS } from 'modules/auction/utils'
+import { auctionParamsType } from 'components/types'
+import AuctionStaticPage from 'components/AuctionStaticPage'
+import SignInNotice from 'components/SignInNotice'
+
+import './AuctionFinishedPage.css'
+
+export default class AuctionFinishedPage extends React.PureComponent {
+  static propTypes = {
+    onGoToMarketplace: PropTypes.func.isRequired,
+    isConnected: PropTypes.bool.isRequired,
+    isConnecting: PropTypes.bool.isRequired,
+    params: auctionParamsType
+  }
+
+  getAuctionRealDuration = () => {
+    const { startTime, endTime } = this.props.params
```

Comment:
get `startTime` from `auction/utils`

Reply:
- Add a `// The transaction that ends the auction might take a while to finish so we cap the max duration`comment
- Move it to utils